app/views/govuk_component/taxonomy_sidebar.raw.html.erb

```diff
@@ -1,20 +1,36 @@
 <% if local_assigns[:items] && !items.blank? %>
   <aside class='govuk-taxonomy-sidebar' data-module='track-click' role='complementary'>
-    <% items.each_with_index do |item, item_index| %>
+    <% items.sort_by { |item| item[:title] }.each_with_index do |item, item_index| %>
```

Comment:
That makes much more sense actually. Will move this behaviour and its tests over.